tests/bluechi_test/container.py

```diff
@@ -63,6 +63,15 @@ def gather_journal_logs(self, data_dir: str) -> None:
 
         self.get_file(log_file, data_dir)
 
+    def gather_coverage(self, data_dir: str) -> None:
+        gcno_file_location = "/usr/share/bluechi-coverage/"
+        gcda_file_location = "/root/rpmbuild/BUILD/bluechi-0.6.0/redhat-linux-build/src"
+
+        result, output = self.exec_run(f"ls -la {gcda_file_location}", tty=True)
```

Comment:
What is the `ls -al` output for? Don't you know the exact location and name of each gcda file?

Reply:
Form what I have found the gcda location is in "/root/rpmbuild/BUILD/bluechi-0.6.0/redhat-linux-build/src" and there is no other path to the gcda. the `ls -la` command is to check if the location exist because in some nodes there is no such location(and no gcda files). Maybe I need a better command to check if dir exist.

Reply:
Hmm, so if  `.gcda` files are created by default in the same location as object files https://splichal.eu/gccsphinx-final/html/gcc/gcov/brief-description-of-gcov-data-files.html then we need additional PR, which would contain:

1. Modify `meson.build` to add `-fprofile-dir=/var/tmp/bluechi-coverage` to compiler when building with coverage created
2. Modify `bluechi.spec.in` which would created `/var/tmp/bluechi-coverage` as a part of `bluechi-coverage` RPM installation

With those changes above you could just gather `.gcda` files from `/var/tmp/bluechi-coverage` without any detection magic